plumber/asset/material.py

```diff
@@ -42,6 +42,8 @@ def import_material(material: Material) -> None:
     if material_data is None:
         material_data = bpy.data.materials.new(material_name)
 
+    material_data['path_id'] = material.name()
```

Comment:
Not sure, but should we have "plumber" in the custom property names somewhere so that it's more clear where they came from? Like "plumber_path_id" etc., or something. Though since plumber imports the whole thing, maybe it's already clear that they came from there as well.

Reply:
this could probably be "path_posix". because string properties have a much larger character limit, i can store where in the game files the materials come from so i can link an existing material with the same path. this helps with recycling

Reply:
Why "_posix" in the name? Sorry, I didn't understand what you meant with that. The current name for the custom property is ok as well.

Reply:
path_posix was a variable tossed around quite often in SourceIO. i thought that it would fit. path_id would be better and is used by my porting tools

Reply:
Ok, that doesn't really make any sense here, but the current name is good